modules/nodes-namespaced-nodelevel-sysctls.adoc

```diff
@@ -21,7 +21,11 @@ Additionally, most of the sysctls in the `net.*` group are known to be namespace
 Sysctls that are not namespaced are called _node-level_ and must be set
 manually by the cluster administrator, either by means of the underlying Linux
 distribution of the nodes, such as by modifying the `_/etc/sysctls.conf_` file,
-or by using a daemon set with privileged containers. You can use the Node Tuning Operator to set _node-level_ sysctls.
+or by using a daemon set with privileged containers. You can use the Node Tuning Operator to set _node-level_ sysctls. 
```

Comment:
Sorry for the revision request.

As per https://redhat-documentation.github.io/supplementary-style-guide/#admonitions, we don't stack admonitions. In the event that we need two, we either 1) restructure content; or 2) use one admonition with multiple bullet points.

For this content, I suggest removing this NOTE and making the following revision in your text:

```suggestion
or by using a daemon set with privileged containers. You can use the Node Tuning Operator to set _node-level_ sysctls. For more information, see "Node Tuning Operator."
```
THEN, in the nodes/containers/nodes-containers-sysctls.adoc assembly, we'll add the link in the additional resources section. This section looks like the following. Copy and paste the second bullet point:

[role="_additional-resources"]
[id="additional-resources_nodes-containers-sysctls"]
== Additional resources

* xref:../../networking/configure-syscontrols-interface-tuning-cni.adoc#nw-configuring-tuning-cni_configure-syscontrols-interface-tuning-cni[Configuring system controls by using the tuning CNI]
* xref:../../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[Node Tuning Operator]

Reply:
@stevsmit Thanks for your input and for the labeling help! @anandrece if you can please make this edit, I'll resubmit for merge. Don't forget to squash commits!